File: Selection Sort/sol.py
# O(n*n) time and O(1) space

def selectionSort(array):
    # Write your code here.
    startInd=0
    while startInd<len(array)-1:
        minInd=startInd
        for i in range(startInd+1,len(array)):
            if array[minInd]>array[i]:
                minInd=i

        array[minInd],array[startInd]=array[startInd],array[minInd]

        startInd+=1
    return array

File: Sorting/Selection Sort/sol.js
// O(n*n) time and O(1) space

function selectionSort(array) {
  // Write your code here.
  let startInd = 0;
  while (startInd < array.length) {
    let minInd = startInd;
    for (let i = startInd + 1; i < array.length; i++) {
      if (array[i] < array[minInd]) {
        minInd = i;
      }
    }

    const temp = array[minInd];
    array[minInd] = array[startInd];
    array[startInd] = temp;

    startInd++;
  }
  return array;
}
